ec_email_client/src/pages/InboxPage.js

```diff
@@ -122,6 +122,12 @@ function InboxPage(props) {
         return atob(data);
     }
 
+    function Test() {
```

Comment:
May get a little confusing naming this Test but I'm not a picky person

Reply:
Yeah this is the only thing I'd have any concerns with. We can change it in future if we need to hurry this PR in though...